<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Kod Dedektifi</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_k">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>155</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Parmak izi  boyutu (k):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_k">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>110</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>511</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser
{
	background: #FFF;
	padding: 10px;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushDir">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>341</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	color: white;
	background: #34495e;
}</string>
    </property>
    <property name="text">
     <string>Seç...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Kaynak konum:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushStart">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>500</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	color: white;
	background: #34495e;
}</string>
    </property>
    <property name="text">
     <string>Hesapla</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Kaynak Kodlar:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>İnce ayarlar:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkTweaks">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>21</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>(Detaylar için rehbere göz atın.)</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_k">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>115</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="pageStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_w">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>155</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>15</number>
    </property>
    <property name="pageStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_w">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>155</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Pencere boyutu (w):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_w">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>150</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDedektif">
    <property name="title">
     <string>Dedektif</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuRehber">
    <property name="title">
     <string>Rehber</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHakk_nda">
    <property name="title">
     <string>Hakkında</string>
    </property>
   </widget>
   <addaction name="menuDedektif"/>
   <addaction name="menuRehber"/>
   <addaction name="menuHakk_nda"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
